frontend-site/src/modules/staticData.js

```diff
@@ -1,29 +1,72 @@
   'Arduino',
+  'LUG API',
+  'Board Games',
+  'Game Systems',
   'Ice Cream',
-  '3-D Printing',
 ]);
 
 export const servicesOffered = Object.freeze([
   // TODO: fill in empty entries
   {
-    name: 'Raspberry Pi',
-    image: '',
-    description: '',
+    name: 'Hardware',
+    image: require("@/assets/Hardware.png"),
+    description: 'From resistors to soldering irons, the LUG is stocked with a wide variety of hardware componants available for student use.',
     url: '',
   },
+  {
+    name: '3D Printing',
+    image: require("@/assets/3DPrinter.png"),
+    description: 'Becoming a LUG member brings many perks such as having easy access to our 3D printer located in the ACM/LUG office. Members can watch their creations come alive for minimal cost.',
+    url: 'https://lug.cs.uic.edu/3dprint/home',
+  },
+  {
+    name: '3-D Scanning',
+    image: require("@/assets/3DScanner.png"),
+    description: 'A 3-D scanner is a device that scans and analyses objects and returns a 3-D 360° image. Available for use in the LUG office, students will have access to a rare 3-D technology, which could be used along side the 3-D scanner.',
+    url: 'https://lug.cs.uic.edu/3dprint/home',
+  },
+  {
+    name: 'Raspberry Pi',
+    image: require("@/assets/RasberryPi.jpg"),
+    description: 'The Raspberry Pi, single-board computer, has been instrumental in encouraging student creativity and LUG is here to help students gain access to one. We offer Raspberry Pi rentals for any student interested. A Raspberry Pi is powerful enough to run a full Linux installation and many student projects; ask an officer if it\'s the right choice for you.',
+    url: '',
+    },
   {
     name: 'Arduino',
-    image: '',
-    description: '',
+    image: require("@/assets/Arduino.jpg"),
+    description: 'Arduino was born to be an easy tool for fast prototyping, aimed at students without a background in electronics and programming. If you are in need of an Arduino for a class or simply want to test out the technology, LUG has you covered.',
     url: '',
   },
   {
-    name: '3D Printing',
-    image: '',
-    description: '',
-    url: 'https://lug.cs.uic.edu/3dprint/home',
+    name: 'LUG API',
+    image: require("@/assets/API.png"),
+    description: 'LUG is determined to present students with tools that will help them on their road to success, which is why we made our API open to anyone, presenting students with the opportunity to see how this technology works.',
+    url: '',
   },
+  {
+    name: 'Board Games',
+    image: require("@/assets/BoardG.jpg"),
+    description: 'Need a break a stressful week of classes? Pick up a board game, located in the LUG office, and grab some friends. There are plenty of games to choose from.',
+    url: '',
+  },
+  {
+    name: 'Game Systems',
+    image: require("@/assets/Sega.png"),
+    description: 'If your feeling nostalgic, you can try out the sega or the various other early gaming systems found in the LUG office.',
```

Comment:
Grammar/Spelling: `If your feeling...` -> `if you're feeling...`

(Minor) `sega` -> `Sega` or `SEGA`

(Minor) `early gaming systems` -> `retro gaming systems` (I'll leave this suggestion up to you to decide on changing it)